released/packages/coq-flocq-quickchick/coq-flocq-quickchick.1.0.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+name: "coq-flocq-quickchick"
+version: "1.0.2"
```

Comment:
Name and version are redundant.

Reply:
Redundant with the file name.

Reply:
**with the file name** you mean, that if I rename `opam` file to `coq-flocq-quickchick.1.0.2.opam` I can omit this fields?

Reply:
Okay, I looked at another package opam file and figured it out myself, thank you